pkgs/development/python-modules/lastversion/default.nix

```diff
@@ -0,0 +1,89 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+  # nativeBuildInputs
+  pythonRelaxDepsHook,
+  setuptools,
+  # nativeCheckInputs
+  pytestCheckHook,
+  # install_requires
+  appdirs,
+  beautifulsoup4,
+  cachecontrol,
+  distro,
+  feedparser,
+  packaging,
+  python-dateutil,
+  pyyaml,
+  requests,
+  tqdm,
+  urllib3,
+}:
+
+buildPythonPackage rec {
+  pname = "lastversion";
+  version = "3.5.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "dvershinin";
+    repo = "lastversion";
+    rev = "v${version}";
+    hash = "sha256-SeDLpMP8cF6CC3qJ6V8dLErl6ihpnl4lHeBkp7jtQgI=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    appdirs
+    beautifulsoup4
+    cachecontrol
+    distro
+    feedparser
+    packaging
+    python-dateutil
+    pyyaml
+    requests
+    tqdm
+    urllib3
+  ] ++ cachecontrol.optional-dependencies.filecache;
+
+  nativeBuildInputs = [ pythonRelaxDepsHook ];
+
+  pythonRelaxDeps = [
+    "cachecontrol" # Use newer cachecontrol that uses filelock instead of lockfile
+    "urllib3" # The cachecontrol and requests incompatibility issue is closed
+  ];
+
+  pythonRemoveDeps = [
+    "lockfile" # "cachecontrol" now uses filelock
+  ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  pytestFlagsArray = [
+    "tests/test_cli.py"
+    "-k"
+    "test_cli_format"
```

Comment:
Please use disabledTests instead

Reply:
The problem is that `test_cli_format` in `tests/test_cli.py` is the only test case that doesn't require an internet connection.

Should I turn off all the tests via `doCheck = false`?